.github/workflows/lifecycle-image.yml

```
@@ -0,0 +1,24 @@
+name: lifecycle-image
+
+on:
+  release:
+    types:
+      - published
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Get release version
+        id: get_release_version
+        run: echo ::set-output name=VERSION::${GITHUB_REF/refs\/tags\/v/}
+      - name: Publish to Registry
+        uses: elgohr/Publish-Docker-Github-Action@master
+        with:
+          name: buildpacksio/lifecycle-image
```

```suggestion
          name: buildpacksio/lifecycle:${{ steps.get_release_version.outputs.VERSION }}
```

1. I don't think there's a need for the name to include "image"
2. We'd probably want versioned tags along with `latest`.

---

+1

---

I'll remove `-image`. We're already getting the versioned tag + latest here: https://github.com/buildpacks/lifecycle/blob/5fa1ed064fc4f032dc32c1586e03939a7342bdfc/.github/workflows/lifecycle-image.yml#L24